circuits/aes-gcm/aes-gcm-foldable.circom

```diff
@@ -1,6 +1,5 @@
 pragma circom 2.1.9;
 
-include "ghash-foldable.circom";
 include "aes/cipher.circom";
```

Comment:
making note to self to annotate difference between files called foldable and fold suffix at module level comment next week